prisma/seed.ts

```diff
@@ -42,6 +45,34 @@ async function main() {
       })
     );
   }
+
+  const allLaborMarkets = await prisma.laborMarket.findMany();
+
+  // create 10 fake service requests/challenges for each labor market in Prisma
+  async function seedServiceRequests(laborMarkets: LaborMarket[]): Promise<ServiceRequest[]> {
+    laborMarkets.forEach((laborMarket) => {
```

Comment:
`forEach` and async/await can lead to quite a bit of confusion. Here you're firing off 10 concurrent requests. Then, without any guarantee that any of them have finished, calling  `serviceRequest.findMany()`.

As a general rule, I like to avoid mixing `forEach` and async/await.
```ts
// wait for each request synchronously. No big deal, not overly concerned with performance
for (const laborMarket of laborMarkets) {
 await upsertServiceRequest(fakeServiceRequest({}, laborMarket.address as string));
}
```

Also, I think there's an opportunity to be more semantic with our seed script. I'm thinking something like this:

```ts
const laborMarkets = seedLaborMarkets();
const serviceRequests = seedServiceRequests(laborMarkets);
seedSubmissions(serviceRequests);
```